### PR TITLE
Added Aqua Security dashboard

### PR DESCRIPTION
**Pull request by jerbia**
This pull request contains a Sumo Logic dashboard for Aqua Security.
It can be used to display activity generated by Aqua Security in a container platform.

### CODE REVIEW

File: AquaSecurity/AquaSecurity-Dashboard.json [hunk 1]
```diff
@@ -0,0 +1,1050 @@
...
+  "description":"",
+  "columns":[""],
+  "detailLevel":0,
+  "properties":"{\"layoutType\":\"gridster\"}",
+  "panels":[{
+    "type":"ReportPanel",
+    "name":"Detected Events",
+    "viewerType":"svv",
+    "detailLevel":2,
+    "queryString":" _sourceCategory=\"Audit\"\n| json field=_raw \"result\" | where %result = 3 | count by result",
```
Comment (kb18951452) [inline AquaSecurity/AquaSecurity-Dashboard.json]: @jerbia Thanks for the PR, 
We are working on ensuring that these are easier/standard for users to deploy. 
Would you make 2 small changes please and i'll merge this PR?
In the JSON, the "scope" is defined as `_sourceCategory=\"Audit\"\n`, which I assume is the sourcecategory you used for the logs this dashboard was created against. 

Would you please change the scope to something like `$$aquaauditlogs` and 

secondly, Would you make a note in the README.md that `$$aquaauditlogs` must be replaced with something like `_sourceCategory=<source_category>\n` where <source_category> is the _sourceCategory for your aqua audit logs`?

File: AquaSecurity/README.md  [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Contains a dashboard for Aqua Security (https://www.aquasec.com)
```
Comment (kb18951452) [inline AquaSecurity/README.md ]: Please add the replace foo (mentioned in the JSON comment) in the README.md

File: AquaSecurity/AquaSecurity-Dashboard.json (hunk 1 above)
Comment (jerbia) [reply AquaSecurity/AquaSecurity-Dashboard.json]: Hi. What is the $$aquaauditlogs? Is it an environment variable? 
How would a user provide a value to this environment variable? Is there some system setting in SumoLogic where you can provide values to environment variables?

**Comment by kb18951452** [comment]
I slightly adjusted the JSON and the README.md. My apologies for not having an up-to-date doc for this.

**Pull request merged**